examples/vae_text/vae_train.py

```diff
@@ -107,14 +103,13 @@ def __init__(self,
                 input_size=(self.decoder_w_embedder.dim +
```

Comment:
I don't think this is right. Now decoders take either `token_embedder` or `token_pos_embedder` as argument (which can be `nn.Module` or a function) in their constructors. Since you also need to concatenate `latent_z`, I would suggest storing `latent_z` as an attribute and writing a method as `token_embedder`.

Reply:
Got it.